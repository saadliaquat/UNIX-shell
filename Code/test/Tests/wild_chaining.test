cat ../src/*.c | grep -o return | wc -l > ret_count.log ; cat ret_count.log ; rm ret_count.log
grep "total" Tests/test_output/*.msh.out | sort -r | uniq > Tests/grep.log && echo "Search successful, logged to grep.log" || echo "Failure" ; false || echo "Print" ; rm Tests/grep.log
echo "One letter ext" && ls ../*/*.? | grep -o '\.[a-zA-Z]$' | uniq | grep -o '[a-zA-Z]'