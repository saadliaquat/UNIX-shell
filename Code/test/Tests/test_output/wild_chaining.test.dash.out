13
Search successful, logged to grep.log
Print
One letter ext
o
h
c
